### ISSUE TITLE
`renku log` to generate `sameAs` triples for a user pointing to his `orcid-id`

### ISSUE THREAD

**Issue by jachro** [SwissDataScienceCenter/renku-python#663]
As a Renku owner, I'd like `renku log` to optionally generate a `schema:sameAs` user triple pointing to its `orcid-id` resource.

**Comment by rokroskar** [comment]
So we should default to always use a renku-specific ID but when the user optionally provides an orcid-id we can add it to the KG.

**Comment by jachro** [comment]
Yes, that was my idea. But now I'm a bit not sure, who should be searching for the `orcid-id': `renku-cli` or `triples-generator`?

**Comment by Panaetius** [comment]
This issue is relevant again with the OLOS export in https://github.com/SwissDataScienceCenter/renku-python/issues/1848 as OLOS uses orcid-ids as the main identifier for users and this would allow us to map users between systems.

I'm not sure `renku-login` is needed for this, we could store a user email <--> orcid map in renku.ini with some command and use that
